Sources/Web3Core/KeystoreManager/IBAN.swift

```diff
@@ -64,9 +64,9 @@ public struct IBAN {
 
     internal static func decodeToInts(_ iban: String) -> String {
         let uppercasedIBAN = iban.replacingOccurrences(of: " ", with: "").uppercased()
-        let begining = String(uppercasedIBAN[0..<4])
+        let beginning = String(uppercasedIBAN[0..<4])
```

Comment:
Please undo this changes, as they affecting code, rather then comments and should be reviewed separately.